pdr_backend/lake/test/test_etl.py

```diff
@@ -189,28 +183,29 @@ def test_drop_temp_sql_tables(setup_data):
     etl, pds, _ = setup_data
 
     # SELECT ALL TABLES FROM DB
-    table_names = pds.get_table_names()
+    original_tables = pds.get_table_names()
+    assert len(original_tables) == 5
```

Comment:
and especially here!
thoughout the implementation, asserts should be in relation to something else in the code, not a hardcoded number

Reply:
yeah, i tried to do it throughout this whole test... i skipped this one because there are default tables in duckdb "main" schema... i'll fix this.